autovacuum_message_attachment/models/ir_attachment.py

```diff
@@ -22,7 +22,7 @@ def _get_autovacuum_domain(self, rule):
             inheriting_model = self.env[rule.inheriting_model]
             attachment_link = inheriting_model._inherits.get("ir.attachment")
             att_ids = inheriting_model.search(create_date_domain).mapped(
-                attachment_link + ".ids"
+                attachment_link + ".id"
```

Comment:
wouldn't be cleaner to do `inheriting_model.search(create_date_domain).mapped("attachment_link").ids`

Reply:
Agree here, it's non-blocking but not a lot of work I guess?

Reply:
I guess, I first did that but I changed to do the minimal change.
I'll update this way, no problem.